tests/src/Kernel/fixtures/markup_rendering_patterns/listing.yml

```
@@ -319,13 +315,15 @@ listing_default_3_col:
   assertions:
     count:
       'div.bcl-listing.bcl-listing--default-3-col': 1
-      'div.row.row-cols-1.row-cols-lg-3.g-2': 1
+      'div.row-cols-md-2.row.row-cols-1.row-cols-xl-3.g-4': 1
       'div.row': 7
       'div.listing-item': 6
       'img.rounded-1': 5
       'p.card-text.mb-3': 6
       'span.me-4': 6
       'span.mb-2': 12
+      'h5.card-title': 6
+      'h4.fw-bold.mb-4': 1
```

Add link attributes testing: 
```suggestion
      'h5.card-title': 4
      'a.d-inline-block': 1

---

Assert also .mw-listing-img' and 'mw-date'
when image or date is displayed, for instance:
``      '.mw-listing-img': 5
on "listing_default_3_col_tag

---

Yes, for this PR it will be done for image and later on, on the corresponding PR, I'll add it for date.